### ISSUE TITLE
DnD: Position der Elemente innerhalb einer Ablegeliste (Regal)

### ISSUE THREAD

**Issue by jkoelm** [iqb-berlin/verona-modules-aspect#304]
Optional sollten Elemente innerhalb der Startablegeliste sollen eine feste Position haben können und nicht nachrutschen wenn eins rausgeommen wird. Beim Zurücklegen in die Liste sollen die Elemente automatisch an ihre Ursprungsposition zurückgehen.

**Comment by rhenck** [comment]
Bin nicht sicher, ob ich das richtig verstanden habe. Grundsätzlich ist aber zu sagen: Es gibt keine "Startablegeliste" - alle Listen sind baugleich. Außerdem trägt ein Element keine Info darüber, wo es irgendwann mal herkam.
Vielleicht gefällt euch das mit den neuen Ablegelisten besser, ansonsten würde ich das Ticket gerne wegdiskutieren... ;)

**Comment by rhenck** [comment]
Ist das sowas die Seebrücke 6? Falls ja, könnt ihr das ja nun und das Ticket kann zu.

**Comment by jkoelm** [comment]
Nein, hier ging es um Listen, in denen die Elemente nicht kopiert werden, wie z. B. hier https://www.iqb-studio.de/#/a/7/2061/preview Teilaufgabe 7 (da haben wir es umständlich in verschiedenen vorbelegten Listen angelegt, damit jedes Element seine Position behält, einfacher wäre aber alles in einer Startliste).

**Comment by jkoelm** [comment]
Wäre gut, wenn das nächste Woche noch umgesetzt werden könnte. Falls es zeitlich nicht mehr passt, könnten wir aber den oben beschriebenen umständlichen Workaround anwenden.

**Comment by rhenck** [comment]
Also das Ding ist, wider Erwarten (zumindest meines), sehr kompliziert. Hier ist der Aufwand sehr hoch für einen mMn sehr kleinen Gewinn und ich würde das ersatzlos streichen. Aber selbst wenn man das anders sieht, würde das länger dauern und birgt die Gefahr neue Bugs zu produzieren, weil man ziemlich tief in den Code rein muss. Da ihr ja schon signalisiert habt, zur Not auch ohne das Feature leben zu können, mache ich da vorerst nichts weiter dran. Okay?

**Comment by jkoelm** [comment]
ja, okay

**Comment by jkoelm** [comment]
Perspektivisch hätten wir das gerne. Wir kommen zwar mit unserem Workaround zurecht, aber es starten jetzt auch immer mehr AE-Gruppen und gerade diese DnD-Sachen sind dann nicht so einfach anzulegen für viele. Eine einzelne Startliste würde es vereinfachen.

**Comment by rhenck** [comment]
Probleme:
- Einzellisten nebeneinander lassen sich nur schwer bauen und bricht nicht dynamisch um
- man kann nicht fertig gebautes einfach auf Einzellisten umstellen
- man kann Elemente in die falsche Startliste zurücklegen
- Durchschriftfunktion nicht möglich
